### ISSUE TITLE
Inclusion request: Macedonian adBlock Filters

### ISSUE THREAD

**Issue by DandelionSprout** [AdguardTeam/FiltersRegistry#354]
I'm of the belief that mainstream adblockers should have at least one list from all significant languages, especially those that have Firefox or Chrome UI translations. I've long kept an eye on the situation among two somewhat conflicting Macedonian lists that by themselves were unfit for any adblockers:

* [Macedonian adBlock Filters](https://raw.githubusercontent.com/RandomAdversary/Macedonian-adBlock-Filters/master/Filters) (2 years since its last update, until tonight.)
* [Macedonian Pi-hole Blocklist](https://raw.githubusercontent.com/cchevy/macedonian-pi-hole-blocklist/master/hosts.txt) (Has no hiding rules.)

I decided tonight to make my move, and combine the latter list into the former. This means that [Macedonian adBlock Filters](https://raw.githubusercontent.com/RandomAdversary/Macedonian-adBlock-Filters/master/Filters) now has 680 entries, and I was positively surprised by [the quick reply time of the repo's owner](https://github.com/DeepSpaceHarbor/Macedonian-adBlock-Filters/pull/2#event-3474373654).

tl;dr: I propose the addition of [Macedonian adBlock Filters](https://raw.githubusercontent.com/RandomAdversary/Macedonian-adBlock-Filters/master/Filters) to AdGuard.

**Comment by DandelionSprout** [comment]
Should this become a success, I also have good eyes at [Albanian](https://raw.githubusercontent.com/AnXh3L0/blocklist/master/albanian-easylist-addition/Albania.txt) and [Maldivian](https://raw.githubusercontent.com/evenxzero/Raajje-AdList/master/filter.txt) lists, who sadly have also had pretty sloppy maintenance in the past year, but which [I've also](https://github.com/AnXh3L0/blocklist/pull/1) [tried to](https://github.com/evenxzero/Raajje-AdList/pull/1) nudge and encourage tonight.

**Comment by zloyden** [comment]
[Macedonian adBlock Filters](https://raw.githubusercontent.com/RandomAdversary/Macedonian-adBlock-Filters/master/Filters) was updated long time ago.
[Macedonian Pi-hole Blocklist](https://raw.githubusercontent.com/cchevy/macedonian-pi-hole-blocklist/master/hosts.txt) should be oriented towards content blockers.

**Comment by ameshkov** [comment]
See the discussion here: https://github.com/AdguardTeam/AdguardFilters/pull/144318

Let's reconsider.
